app/main/routes.py

```diff
@@ -52,7 +52,7 @@ def upload():
 
         current_app.logger.info("POST sent to data-store /ingest")
         success, pre_errors, validation_errors, metadata = post_ingest(
-            excel_file, {"reporting_round": 4, "place_names": place_names}
+            excel_file, {"reporting_round": 4, "place_names": place_names, "do_load": not Config.DISABLE_LOAD}
```

Comment:
I suppose this is the corollary of writing the variable as ```do_load``` - it sounds more like loading is disabled by default and then a flag is set to enable it to load. This would probably therefore read cleaner is it was something like  ```Config.ALLOW_load```, with the env var set to True as a default.

Reply:
I agree this would read better if it were `ALLOW_LOAD`. 

We'd then have to set the setup to:
```
ALLOW_LOAD = not os.environ.get("DISABLE_LOAD", False)
```
and so just pass it upstream. I think this is less clear.

One option is to create a utils function like:
```
def is_load_enabled():
   return not Config.DISABLE_LOAD
```

Reply:
I've implemented this utils function for clarity